py/src/braintrust/util.py

```diff
@@ -47,5 +47,9 @@ def get_caller_location():
         if this_dir is None:
             this_dir = os.path.dirname(caller.filename)
         if os.path.dirname(caller.filename) != this_dir:
-            return dict(caller_filename=caller.filename, caller_lineno=caller.lineno)
-    return {}
+            return dict(
+                caller_function_name=caller.function,
+                caller_file_name=caller.filename,
+                caller_line_num=caller.lineno,
```

Comment:
I don't think we shold rename these -- we already have people who have logged them, and we'll be in a permanently inconsistent state. They're hardcoded to be ignored in the trace UI implementation right now.